player/js/elements/svgElements/SVGEffects.js

```diff
@@ -33,7 +33,7 @@ function SVGEffects(elem){
     }
     if(count){
         elem.globalData.defs.appendChild(fil);
-        elem.layerElement.setAttribute('filter','url(#'+filId+')');
+        elem.layerElement.setAttribute('filter','url(' + location.href + '#'+filId+')');
```

Comment:
@goel4ever please feel free post your test app on a code pen, so people can see the issue easily.